scripts/check_release_branch_date.py

```diff
@@ -0,0 +1,27 @@
+import json
+from datetime import datetime, date, timedelta
+from urllib import request
+
+from common import get_patch_version, env
+
+if __name__ == '__main__':
+    repo = env("GITHUB_REPOSITORY")
+    response = request.urlopen(f"https://api.github.com/repos/{repo}/milestones")
```

Comment:
I don't use `request` or another more convenient library on purpose not to complicate python setup on CI for single HTTP request

Reply:
Fair enough 👍